doc/version_history.rst

```diff
@@ -1,11 +1,32 @@
-.. py:currentmodule:: lsst.ts.ess
+.. py:currentmodule:: lsst.ts.ess.csc
 
-.. _lsst.ts.ess.version_history:
+.. _lsst.ts.ess.csc-version_history:
 
 ###############
 Version History
 ###############
 
+v0.7.0
+======
+
+* Modify to use data clients (instances of `lsst.ts.ess.common.BaseDataClient`) to communicate with data servers.
+  This requires ts_ess_common 0.7.
+* Use new error codes from ts_idl 3.7, which is recommended but not required, due to a temporary local version of the ErrorCode enum class.
+  All clients of this CSC should use ts_idl v3.7.0 in order to get correct ErrorCode values.
+* Fix API docs.
+* Enable mypy type checking.
+* Change ``master`` to ``main`` in CONFIG_SCHEMA's ``id``, in preparation for renaming the branch.
+
+Requires:
+
+* ts_salobj 6.3
+* ts_idl 3.7
```

Comment:
This version of ts_idl is not required, right?

Reply:
Good point. I modified the text to recommend 3.7 but allow 3.5 and 3.6